evm/src/executor/inspector/cheatcodes/ext.rs

```diff
@@ -189,6 +189,15 @@ fn get_env(key: &str, r#type: ParamType, delim: Option<&str>) -> Result<Bytes, B
     value_to_abi(val, r#type, is_array)
 }
 
+fn project_root(state: &Cheatcodes) -> Result<Bytes, Bytes> {
+    let root =
+        state.config.root.clone().into_os_string().into_string().map_err(|_| {
+            util::encode_error("invalid characters in project root path".to_string())
+        })?;
```

Comment:
we should just assume valid paths and simply do `let root = format!("{}", state.config.root.display())`

Reply:
sounds good, updated in d235f827b604642286fca34fe64e7c99786f8826